Main-rpn/Main-rpn/Main-rpn.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <fstream>
+
+#include <math.h>
+#include <string>
+#include <stack>
+
+int calculate(const std::string &infix)
+{
+    std::cout << infix << "\n";
+    std::string postfix = "";
+
+    std::stack<char> operators;
+
+    // Convert from infix to postfix notation
+    for (int i = 0; i < infix.size(); i++)
+    {
+        if (infix[i] >= 48 && infix[i] <= 57)
+        {
+            postfix += infix[i];
+            postfix += ' ';
+        }
+        else if (infix[i] == '+' || infix[i] == '-' || infix[i] == '*' || infix[i] == '/' || infix[i] == '^')
+            operators.push(infix[i]);
+        if (infix[i] == ')' || i == infix.size() - 1)
+            if (!operators.empty())
+            {
+                postfix += operators.top();
+                postfix += ' ';
+                operators.pop();
+            }  
+    }
+
+    // Calculate the result from the postfix notation
+    // Assume all numbers are 1-digit 
+
+    std::stack<int> operands;
+
+    for (int i = 0; i < postfix.size(); i++)
+    {
+        if (postfix[i] >= 48 && postfix[i] <= 57)
+            operands.push(int(postfix[i]) - 48);
+        if (postfix[i] == '+' || postfix[i] == '-' || postfix[i] == '*' || postfix[i] == '/' || postfix[i] == '^')
+        {
+            int firstNum;
```

Comment:
When declaring a variable, it should be assigned a value. https://cwe.mitre.org/data/definitions/457